src/controllers/itemDetails/index.js

```diff
@@ -1894,6 +1895,10 @@ import 'emby-select';
             playCurrentItem(this, this.getAttribute('data-mode'));
         }
 
+        function onPosterClick(e) {
+            itemShortcuts.onClick.call(view.querySelector('.detailImageContainer'), e);
```

Comment:
Wouldn't you be able to use `this` instead of querying for the element again?

Reply:
It's to be explicit as to what the container is, since `itemShortcuts.onClick` is usually used on viewContainers and seems to expect the root item to be a container.